apps/form-builder/src/components/MultipleChoiceQuestionAnalytics/Index.tsx

```diff
@@ -61,7 +61,8 @@ export default function MultipleChoiceQuestionAnalytics({
   return (
     <>
       <Typography variant="h6">
-        {question.questionType}
+        {'Choice Type: '}
+        {question.multipleChoiceOptions?.choiceType}
         {" 'Other' responses"}
```

Comment:
change this to just be "Choice Type: [question.multipleChoiceOptions?.choiceType]" and you're all good!